### PR TITLE
DOC: Update README file.

### PR DESCRIPTION
**Pull request by hhslepicka**
- Remove badges for Travis, AppVeyor and Landscape (💀)
- Add badges for:

1. Azure Pipelines
2. Anaconda for pydm-dev channel
3. Anaconda for pydm-tag channel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-[![Build Status](https://travis-ci.org/slaclab/pydm.svg?branch=master)](https://travis-ci.org/slaclab/pydm) 
-[![Build status](https://ci.appveyor.com/api/projects/status/sw8rp4an7o79m833?svg=true)](https://ci.appveyor.com/project/hhslepicka/pydm-fxpsx)
-[![Code Health](https://landscape.io/github/slaclab/pydm/master/landscape.svg?style=flat)](https://landscape.io/github/slaclab/pydm/master) 
+[![Build Status](https://dev.azure.com/hhslepicka/pydm-test/_apis/build/status/PyDM%20Pipeline?branchName=master)](https://dev.azure.com/hhslepicka/pydm-test/_build/latest?definitionId=2&branchName=master)
```
Comment (klauer) [inline README.md]: `https://dev.azure.com/hhslepicka` - should the pydm stuff move to a new azure organization?

Comment (hhslepicka) [reply README.md]: Hummmm maybe I copied the wrong link. Let me check

Comment (hhslepicka) [reply README.md]: Great catch! I had both orgs open and I ended up copying from the wrong one.

**Comment by hhslepicka** [comment]
There are some clear issues with a couple of the tests that I will address with this PR in the following commits.

**Pull request merged**